src/TypeScript/TsConfig/TsConfigProvider.cs

```diff
@@ -55,7 +56,9 @@ public TsConfigProvider(ITsConfigsLocator tsConfigsLocator, ITypeScriptEslintBri
 
         public async Task<string> GetConfigForFile(string sourceFilePath, CancellationToken cancellationToken)
         {
-            var allTsConfigsFilePaths = tsConfigsLocator.Locate();
+            var allTsConfigsFilePaths = tsConfigsLocator.Locate(sourceFilePath);
+
+            logger.WriteLine(Resources.INFO_FoundTsConfigs, string.Join(Path.PathSeparator.ToString(), allTsConfigsFilePaths));
```

Comment:
Verbosity - do we want to print this list on every analysis?
Perhaps we should drop it to DEBUG. I think printing the tsconfig that is actually being used each time is ok, but maybe not the whole list.